tools/2d_simple_filter/filter.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0"?>
+<tool name="Filter Image" id="ip_filter_standard" version="0.0.2">
+   <description>Filter Image</description>
+   <requirements>
+        <requirement type="package" version="0.14.0">scikit-image</requirement>
+        <requirement type="package" version="1.15.4">numpy</requirement>
+        <requirement type="package" version="5.3.0">pillow</requirement>
+        <requirement type="package" version="0.15.1">tifffile</requirement>
+        <!-- <requirement type="package" version="1.1.0">scipy</requirement> -->
+   </requirements>
+   <stdio>

```

Comment:
This can be removed entirely and replaced by `detect_errors=` in the command tag.